static/js/impala/addon_details.js

```diff
@@ -1,5 +1,9 @@
 $(function () {
     if (!$("body").hasClass('addon-details')) return;
+    $('#background-wrapper').css('height',
```

Comment:
Should this only run on a restyled page? Maybe check for a `restyle` class on `<body>` or something?